### ISSUE TITLE
Very high CPU usage in both 5.15.2 and 6.4.2 versions.

### ISSUE THREAD

**Issue by thenicegirl** [xiaoyifang/goldendict#333]
I tried both versions, they make my fan rampage. the CPU usage surpasses 60% easily and sometimes above 80%. This app means too much to me, please fix it. @xiaoyifang 
[5.15.2-GoldenDict.exe_windows-2019_20230205.zip](https://github.com/xiaoyifang/goldendict/releases/download/v23.02.06-LanternFestival.230205.9b8f85ca/5.15.2-GoldenDict.exe_windows-2019_20230205.zip) 
[6.4.2-GoldenDict.exe_windows-2019_20230205.zip](https://github.com/xiaoyifang/goldendict/releases/download/v23.02.06-LanternFestival.230205.9b8f85ca/6.4.2-GoldenDict.exe_windows-2019_20230205.zip)
![Snipaste_2023-02-07_03-08-28](https://user-images.githubusercontent.com/53411757/217067175-229de7cd-abc0-4298-ab65-2c2550e0042e.png)

**Comment by xiaoyifang** [comment]
can you enable log and upload the log file
`goldendict.exe --log-to-file --show-error-file`

**Comment by xiaoyifang** [comment]
any fullindex has been in progress? try to disable fulltext and have a try.

**Comment by thenicegirl** [comment]
> can you enable log and upload the log file `goldendict.exe --log-to-file --show-error-file`
> 
> and what's the operation you are doing when this occured. the os platform ,Windows 11?

I executed the command you indicates, but I don't know where the log file is. this is my os platform:
```
Edition	Windows 11 Pro
Version	22H2
Installed on	‎11/‎23/‎2022
OS build	22621.1105
Experience	Windows Feature Experience Pack 1000.22638.1000.0
```

**Comment by thenicegirl** [comment]
> any fulltext index has been in progress? try to disable fulltext and have a try.

I disabled it as you indicate, it performs a bit better but still makes my fan rampage. CPU usage is still about 30% to 50%.

**Comment by BEST8OY** [comment]
> but I don't know where the log file is.

It's here.

"C:\Users\****\AppData\Roaming\GoldenDict\gd_log.txt"

**Comment by thenicegirl** [comment]
> > but I don't know where the log file is.
> 
> It's here.
> 
> "C:\Users\XXXX\AppData\Roaming\GoldenDict\gd_log.txt"

Thanks for telling me. 🙂

[gd_log.txt](https://github.com/xiaoyifang/goldendict/files/10674567/gd_log.txt)
Now I upload it, hope it will help you @xiaoyifang improve.

**Comment by xiaoyifang** [comment]
I checked the log file,it looked ok,nothing valuable in the log file.
I guess the cpu performance may has relation with the Windows 11+Qt.

**Comment by 429IhD0** [comment]
I can confirm this also, but I'm on Win10. 

However, this problem doesn't seem to specific to this fork. On my machine whenever I try to use GD built with qt5 or qt6 to access wikipedia, the fan goes rampage. The problem exists both in main branch and in this fork.

The qt4 user wikipedia experience is so much better (smoother scrolling, page rendering much faster, no noisy fans) that I really wish this fork supports qt4 as well

**Comment by xiaoyifang** [comment]
> The qt4 user wikipedia experience is so much better (smoother scrolling, page rendering much faster, no noisy fans)

maybe the difference between webengine and webkit .  But  webengine is involving and webkit is dead.
qt4 will not be supported ,   It's not in plan.

**Comment by xiaoyifang** [comment]
@thenicegirl  can you create a dump file of this process when the cpu is high. compress the dumped file and upload it here .

**Comment by xiaoyifang** [comment]
> it's audible, but only repeatedly plays the first word I searched, the other ones after the first one play the same sound, whatever I click every speaker icons on the page, it always plays the same.

seems a bug ,can you open a new issue to track this .

**Comment by xiaoyifang** [comment]
> You had known the FFmpeg not working at 6.4.2 #331,

fixed with qt 6.4.3
